widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.kt

```diff
@@ -173,10 +173,29 @@ object GliaWidgets {
                         onComplete.onComplete()
                     } else {
                         Logger.i(TAG, "Glia Widgets SDK initialization failed")
-                        val invalidInputError = GliaWidgetsException(
-                            "Failed to initialise Glia Widgets SDK. Please check credentials.",
-                            GliaWidgetsException.Cause.INVALID_INPUT
-                        )
+                        val invalidInputError = when (exception.cause) {
+                            GliaException.Cause.NETWORK_TIMEOUT -> {
+                                GliaWidgetsException(
+                                    "Network timeout. Please check the Internet connection.",
+                                    GliaWidgetsException.Cause.NETWORK_TIMEOUT
+                                )
+                            }
+
+                            GliaException.Cause.INVALID_INPUT,
+                            GliaException.Cause.FORBIDDEN -> {
```

Comment:
Could you please explain why you decided to add `GliaException.Cause.FORBIDDEN`?